pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -191,9 +191,7 @@ func runArgsForNode(node *config.Node, clusterIPFamily config.ClusterIPFamily, n
 		// suid: SUID applications on the volume will be able to change their privilege
 		// exec: executables on the volume will be able to executed within the container
 		// dev: devices on the volume will be able to be used by processes within the container
-		"--volume", fmt.Sprintf("%s:/var/lib/containerd:suid,exec,dev", volume),
-		"--volume", fmt.Sprintf("%s:/var/lib/kubelet:suid,exec,dev", volume),
-		"--volume", fmt.Sprintf("%s:/var/log:suid,exec,dev", volume),
+		"--volume", fmt.Sprintf("%s:/var:suid,exec,dev", volume),
```

Comment:
i'm slightly confused about this change.
instead of mounting separate directories under /var we now mount /var as a whole, which seem fine, but i don't understand what actual problem with podman we are fixing.

if podman is not accepting multiple --volume arguments properly this seems like a podman bug?

- for the infinite loop your are seeing, is there an indication of what is the actual problem?
- is this reproducible with different podman versions?